contents/handbook/people/team-structure/core-analytics.md

```diff
@@ -11,17 +11,25 @@ hideAnchor: true
 
 ## Mission
 
-Make PostHog the best and most performant product analytics platform.
+Enable any user to painlessly, confidently, and consistently surface insights from their data that are useful for their product
+## Goals & Key Results
 
-## Responsibilities
+- Consistency - Maintain performance and availability of performing analysis
+- Confidence - Guarantee data quality 
+- Painlessness - Ensure features provide direct abilities to analyze their data. Users should not have to create unnecessary workarounds
 
-- Make sure data is correct and reliable
-- Make sure queries are quick
-- Increase the types of queries customers can do in PostHog (including frontend changes)
+## Scope
```

Comment:
To discuss: One of the things that I think there's a lot of confusion with is on the scope of the UX of analytics features. Example: should creating an A+ UI for experimentation be on the scope of this team or Core Experience. Is there a hand-off at some point? Is it a collaborative work?

Reply:
To discuss: Who owns feature flags and why?

Reply:
To discuss: Should Core Analytics be involved with the company's extensibility goal? Connecting PostHog to other systems.